### ISSUE TITLE
Set node count can throw AttributeError 

### ISSUE THREAD

**Issue by cket** [BD2KGenomics/toil#1561]
http://jenkins.cgcloud.info/job/toil-pull-request-integration-tests/987/testReport/junit/src.toil.test.provisioners.aws.awsProvisionerTest/AWSAutoscaleTest/testSpotAutoScale/

**Comment by diekhans** [comment]
That doesn't mean it isn't still a bug.

**Comment by cricketsloan** [comment]
Agreed that it could still be a bug.  But given that large chunks of the code are rewritten, it would need to be reproduced. Do you have evidence that it still exists?

**Comment by diekhans** [comment]
I don't necessary disagree with closing, only the justification
that it is a year stale from someone who is gone.

I don't know if the related code has been modified or not.
Suggest just ask Lon if he suspects the code has been change and
the problem would have been fixed.

Given there is no test case with the bug, `appears to be no
longer a problem' is great by me!  If it is a bug, it will come
back.
